nss-cohort-36/keahua-arboretum-net-surfers
# Jl biome actions

# Description

annex biome list displays correctly and appends biome to list.  also added __init__ to actions for ease of import statements

Closes #[Enter issue #]

## Type of change

Please delete options that are not relevant.

- [x ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)


# Testing Instructions

Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout <branch-name>`
4. in index.py do a print statement on whatever biome you will be created


# Checklist:

- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
